androidApp/src/main/java/com/example/tasky/android/common/presentation/viewmodel/MainViewModel.kt

```diff
@@ -30,7 +30,7 @@ class MainViewModel(
                     bootReceiverManager.stop()
                     syncAgendaManager.stopPeriodicSyncAgenda()
                 }
-            }.stateIn(scope = viewModelScope, started = SharingStarted.Eagerly, initialValue = false)
+            }.stateIn(scope = viewModelScope, started = SharingStarted.Lazily, initialValue = true)
```

Comment:
I would just be careful with creating that coroutine scope in the stateIn block with SharingStarted.Eagerly or SharingStarted.Lazily. Both will effectively keep on listening for changes even if there is no collector that might care about these. You’d be keeping the state in memory even when it is not needed (here it is a small state, so not a big concern)

You could make use of SharingStarted.WhileSubscribed to only cache this when necessary